cppcache/integration-test/CMakeLists.txt

```diff
@@ -205,7 +205,7 @@ set_tests_properties(
     testThinClientPoolAttrTest
     testThinClientPoolLocator
     testThinClientPoolRedundancy
-    testThinClientSecurityAuthentication
+    #testThinClientSecurityAuthentication
```

Comment:
Delete?

Reply:
Actually forgot to uncomment this. I was trying to see if fixing getXmlPath() would fix this test. It doesn't, so I'll disable this test again.